ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -56,6 +56,9 @@ images:
       as:
       - registry.access.redhat.com/ubi8/ubi:latest
   to: e2e-test
+promotion:
```

Comment:
We should either use
```
  additional_images:
    deploy: cincinnati
```
https://docs.ci.openshift.org/docs/getting-started/examples/#publishing-an-image-for-reuse

or simply `s/deploy/cincinnati/g` here

Reply:
I've added `excluded_images` with a2cc988039 -> e7b890bfc1, so now this doesn't add any mirroring entries.